### PR TITLE
AOM-86: Create a notification message when a module starts/stop

### PR DESCRIPTION
**Pull request by Charpell**
### SUMMARY:
Presently, when a start/stop action is successful the only visual cues are a status change from red to green and change of start button to stop button or vice versa. This ticket is to create a notification message when the actions have happened.

### CODE REVIEW

File: app/js/components/manageApps/Addon.jsx [hunk 1]
```diff
@@ -17,7 +17,10 @@ class Addon extends Component {
       starting: false,
       isOpen: false,
       showMessageDetail: false,
-      loadingComplete: false
+      loadingComplete: false,
+      messageBody: '',
+      messageType: 'success',
+      showMessage: false
```
Comment (mkiterian) [inline app/js/components/manageApps/Addon.jsx]: It is good practice to include a trailing comma

File: app/js/components/manageApps/Addon.jsx [hunk 2]
```diff
@@ -243,10 +253,20 @@ class Addon extends Component {
   }
 
   render() {
-    const { app, isOpen, showMessageDetail, affectedModules, action, loadingComplete } = this.state;
+    const { app, isOpen, showMessageDetail, affectedModules, action, loadingComplete,messageBody,
```
Comment (mkiterian) [inline app/js/components/manageApps/Addon.jsx]: List each property in its own line to provide better formatting

**Comment by mkiterian** [comment]
Good job 👍

**Comment by patlub** [comment]
Looks good to me.

**Review by tduyile04 — COMMENTED** [review]
Looks good to me. Great job!!!

**Comment by Farhanisto** [comment]
good job

**Comment by justmesam** [comment]
looks good 👍

**Pull request merged**